### ISSUE TITLE
Pytorch Custom Models (PTH) upscaling incorrectly

### ISSUE THREAD

**Issue by reginaldbuxley** [upscayl/upscayl#605]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.
- [X] I have tried [setting my GPU ID](https://github.com/upscayl/upscayl/wiki/Guide#gpu-id).
- [ ] I am using Windows and I've tried [common troubleshooting steps](https://github.com/upscayl/upscayl/wiki/Troubleshooting).

### Describe the Bug

I have used the tutorial to convert two Pytorch models from openmodeldb.info to NCNN. Both models, on multiple images, produce an image that appears to have many tiles of the image in different color variants (CMYK maybe? I do not know). I have included a screen shot to show this. I could not find out how to address this issue through the issues or discussions pages.

### To Reproduce

Download a Pytorch model on openmodeldb.info (one example I used was @ https://openmodeldb.info/models/2x-SwatKats) and then follow the tutorial @  https://github.com/upscayl/upscayl/wiki/%F0%9F%96%A5%EF%B8%8F-Model-Conversion%E2%80%89%E2%80%93%E2%80%89Get-more-models!

After converting the model, updating the params file and installing your custom model to your custom models folder, attempt an upscale with the model.

Upscayl version == 2.9.5
ChaiNNer version == 0.20.2

### Version (or commit hash)

2.9.5

### Select your Operating System

Linux

### OS Version

Linux Mint 21

### GPU Name

NVIDIA GeForce GTX 970

### Expected Behavior

To have an upscaled image that looks 'normal' for whatever the model does. It should not look like a jigsaw puzzle of different color channels.

### Screenshots

The issue: 
![image](https://github.com/upscayl/upscayl/assets/24417317/5e7603e1-8d7b-4f7d-b2ba-60d30f27189f)

Me showing my GPU is set to the NVIDIA GPU

![image](https://github.com/upscayl/upscayl/assets/24417317/5b9c6fc7-c595-43d8-9d1c-7a0409523696)

### Paste the logs

```sh
🎯 GET_MODELS_LIST:  /home/graphix/models
🐞 BACKEND REPORTED:  📁 Updating Custom Models Folder Path:  /home/graphix/models
🐞 BACKEND REPORTED:  📁 Custom Models Folder Path:  /home/graphix/models
🐞 BACKEND REPORTED:  🔎 Detected Custom Models:  SwatKats
📜 CUSTOM_MODEL_FILES_LIST:  SwatKats
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"SwatKats","value":"SwatKats"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🔀 Setting model to SwatKats
🐞 BACKEND REPORTED:  💾 Updating Save Output Folder:  false
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
🐞 BACKEND REPORTED:  🖼️ Updating No Image Processing:  true
🔄 Resetting image paths
🐞 BACKEND REPORTED:  🖼️ Updating Image Path:  /home/graphix/Downloads/image(3).png
🐞 BACKEND REPORTED:  📄 Selected File Path:  /home/graphix/Downloads/image(3).png
🖼 Selected Image Path:  /home/graphix/Downloads/image(3).png
📁 Selected Image Directory:  /home/graphix/Downloads
🖼 imagePath:  /home/graphix/Downloads/image(3).png
🔤 Extension:  png
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
🐞 BACKEND REPORTED:  🖼️ Updating No Image Processing:  true
🐞 BACKEND REPORTED:  📐 Updating Compression:  0
🐞 BACKEND REPORTED:  ✅ Upscayl Variables:  {"model":"SwatKats","gpuId":null,"saveImageAs":"png","inputDir":"/home/graphix/Downloads","outputDir":"/home/graphix/Downloads","fullfileName":"image(3).png","fileName":"image(3)","initialScale":"4","desiredScale":"4","outFile":"/home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png","compression":0}
🐞 BACKEND REPORTED:  📢 Upscayl Command:  -i,/home/graphix/Downloads/image(3).png,-o,/home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png,-s,4,-m,/home/graphix/models,-n,SwatKats,,,-f,png
🐞 BACKEND REPORTED:  👶 Updating Child Processes:  {"binary":"/opt/Upscayl/resources/bin/upscayl-bin","args":["/opt/Upscayl/resources/bin/upscayl-bin","-i","/home/graphix/Downloads/image(3).png","-o","/home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png","-s","4","-m","/home/graphix/models","-n","SwatKats","","","-f","png"]}
🐞 BACKEND REPORTED:  🛑 Updating Stopped:  false
🐞 BACKEND REPORTED:  image upscayl:  MESA-INTEL: warning: Haswell Vulkan support is incomplete

🚧 UPSCAYL_PROGRESS:  MESA-INTEL: warning: Haswell Vulkan support is incomplete

🐞 BACKEND REPORTED:  image upscayl:  [0 Intel(R) HD Graphics 4600 (HSW GT2)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  fp16-p/s/a=1/0/0  int8-p/s/a=1/0/0
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 Intel(R) HD Graphics 4600 (HSW GT2)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  fp16-p/s/a=1/0/0  int8-p/s/a=1/0/0
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  [1 NVIDIA GeForce GTX 970]  queueC=0[16]  queueG=0[16]  queueT=1[2]
[1 NVIDIA GeForce GTX 970]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 NVIDIA GeForce GTX 970]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/1
[1 NVIDIA GeForce GTX 970]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [1 NVIDIA GeForce GTX 970]  queueC=0[16]  queueG=0[16]  queueT=1[2]
[1 NVIDIA GeForce GTX 970]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 NVIDIA GeForce GTX 970]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/1
[1 NVIDIA GeForce GTX 970]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  [2 llvmpipe (LLVM 15.0.7, 256 bits)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  subgroup=8  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [2 llvmpipe (LLVM 15.0.7, 256 bits)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  subgroup=8  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  0.00%

🚧 UPSCAYL_PROGRESS:  0.00%

🐞 BACKEND REPORTED:  image upscayl:  5.00%

🚧 UPSCAYL_PROGRESS:  5.00%

🐞 BACKEND REPORTED:  image upscayl:  10.00%

🚧 UPSCAYL_PROGRESS:  10.00%

🐞 BACKEND REPORTED:  image upscayl:  15.00%

🚧 UPSCAYL_PROGRESS:  15.00%

🐞 BACKEND REPORTED:  image upscayl:  20.00%

🚧 UPSCAYL_PROGRESS:  20.00%

🐞 BACKEND REPORTED:  image upscayl:  25.00%

🚧 UPSCAYL_PROGRESS:  25.00%

🐞 BACKEND REPORTED:  image upscayl:  30.00%

🚧 UPSCAYL_PROGRESS:  30.00%

🐞 BACKEND REPORTED:  image upscayl:  35.00%

🚧 UPSCAYL_PROGRESS:  35.00%

🐞 BACKEND REPORTED:  image upscayl:  40.00%

🚧 UPSCAYL_PROGRESS:  40.00%

🐞 BACKEND REPORTED:  image upscayl:  45.00%

🚧 UPSCAYL_PROGRESS:  45.00%

🐞 BACKEND REPORTED:  image upscayl:  50.00%

🚧 UPSCAYL_PROGRESS:  50.00%

🐞 BACKEND REPORTED:  image upscayl:  55.00%

🚧 UPSCAYL_PROGRESS:  55.00%

🐞 BACKEND REPORTED:  image upscayl:  60.00%

🚧 UPSCAYL_PROGRESS:  60.00%

🐞 BACKEND REPORTED:  image upscayl:  65.00%

🚧 UPSCAYL_PROGRESS:  65.00%

🐞 BACKEND REPORTED:  image upscayl:  70.00%

🚧 UPSCAYL_PROGRESS:  70.00%

🐞 BACKEND REPORTED:  image upscayl:  75.00%

🚧 UPSCAYL_PROGRESS:  75.00%

🐞 BACKEND REPORTED:  image upscayl:  80.00%

🚧 UPSCAYL_PROGRESS:  80.00%

🐞 BACKEND REPORTED:  image upscayl:  85.00%

🚧 UPSCAYL_PROGRESS:  85.00%

🐞 BACKEND REPORTED:  image upscayl:  90.00%

🚧 UPSCAYL_PROGRESS:  90.00%

🐞 BACKEND REPORTED:  image upscayl:  95.00%

🚧 UPSCAYL_PROGRESS:  95.00%

🐞 BACKEND REPORTED:  image upscayl:  Upscayl Successful

🚧 UPSCAYL_PROGRESS:  Upscayl Successful

🐞 BACKEND REPORTED:  💯 Done upscaling
🐞 BACKEND REPORTED:  ♻ Scaling and converting now...
🐞 BACKEND REPORTED:  🚫 Skipping scaling and converting
upscaledImagePath:  /home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png
💯 UPSCAYL_DONE:  /home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"SwatKats","value":"SwatKats"}
⚙️ Setting gpuId to empty string
⚙️ Getting rememberOutputFolder from localStorage:  false
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"SwatKats","value":"SwatKats"}
⚙️ Getting gpuId from localStorage:  1
🔀 Setting model to 
🔀 Setting model to SwatKats
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
🐞 BACKEND REPORTED:  🖼️ Updating No Image Processing:  true
🐞 BACKEND REPORTED:  📐 Updating Compression:  0
🐞 BACKEND REPORTED:  ✅ Already upscayled at:  /home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png
upscaledImagePath:  /home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png
💯 UPSCAYL_DONE:  /home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png
🔄 Resetting image paths
🐞 BACKEND REPORTED:  🖼️ Updating Image Path:  /home/graphix/Downloads/3e7sbl2v59ea1.jpg
🐞 BACKEND REPORTED:  📄 Selected File Path:  /home/graphix/Downloads/3e7sbl2v59ea1.jpg
🖼 Selected Image Path:  /home/graphix/Downloads/3e7sbl2v59ea1.jpg
📁 Selected Image Directory:  /home/graphix/Downloads
🖼 imagePath:  /home/graphix/Downloads/3e7sbl2v59ea1.jpg
🔤 Extension:  jpg
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🐞 BACKEND REPORTED:  📝 Updating Overwrite:  false
🐞 BACKEND REPORTED:  🖼️ Updating No Image Processing:  true
🐞 BACKEND REPORTED:  📐 Updating Compression:  0
🐞 BACKEND REPORTED:  ✅ Upscayl Variables:  {"model":"SwatKats","gpuId":"1","saveImageAs":"png","inputDir":"/home/graphix/Downloads","outputDir":"/home/graphix/Downloads","fullfileName":"3e7sbl2v59ea1.jpg","fileName":"3e7sbl2v59ea1","initialScale":"4","desiredScale":"4","outFile":"/home/graphix/Downloads/3e7sbl2v59ea1_upscayl_4x_SwatKats.png","compression":0}
🐞 BACKEND REPORTED:  📢 Upscayl Command:  -i,/home/graphix/Downloads/3e7sbl2v59ea1.jpg,-o,/home/graphix/Downloads/3e7sbl2v59ea1_upscayl_4x_SwatKats.png,-s,4,-m,/home/graphix/models,-n,SwatKats,-g,1,-f,png
🐞 BACKEND REPORTED:  👶 Updating Child Processes:  {"binary":"/opt/Upscayl/resources/bin/upscayl-bin","args":["/opt/Upscayl/resources/bin/upscayl-bin","-i","/home/graphix/Downloads/image(3).png","-o","/home/graphix/Downloads/image(3)_upscayl_4x_SwatKats.png","-s","4","-m","/home/graphix/models","-n","SwatKats","","","-f","png"]}
🐞 BACKEND REPORTED:  🛑 Updating Stopped:  false
🐞 BACKEND REPORTED:  image upscayl:  MESA-INTEL: warning: Haswell Vulkan support is incomplete

🚧 UPSCAYL_PROGRESS:  MESA-INTEL: warning: Haswell Vulkan support is incomplete

🐞 BACKEND REPORTED:  image upscayl:  [0 Intel(R) HD Graphics 4600 (HSW GT2)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  fp16-p/s/a=1/0/0  int8-p/s/a=1/0/0
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 Intel(R) HD Graphics 4600 (HSW GT2)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  fp16-p/s/a=1/0/0  int8-p/s/a=1/0/0
[0 Intel(R) HD Graphics 4600 (HSW GT2)]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  [1 NVIDIA GeForce GTX 970]  queueC=0[16]  queueG=0[16]  queueT=1[2]
[1 NVIDIA GeForce GTX 970]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 NVIDIA GeForce GTX 970]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/1
[1 NVIDIA GeForce GTX 970]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [1 NVIDIA GeForce GTX 970]  queueC=0[16]  queueG=0[16]  queueT=1[2]
[1 NVIDIA GeForce GTX 970]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 NVIDIA GeForce GTX 970]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/1
[1 NVIDIA GeForce GTX 970]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  [2 llvmpipe (LLVM 15.0.7, 256 bits)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  subgroup=8  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [2 llvmpipe (LLVM 15.0.7, 256 bits)]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[2 llvmpipe (LLVM 15.0.7, 256 bits)]  subgroup=8  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  0.00%

🚧 UPSCAYL_PROGRESS:  0.00%

🐞 BACKEND REPORTED:  image upscayl:  2.78%

🚧 UPSCAYL_PROGRESS:  2.78%

🐞 BACKEND REPORTED:  image upscayl:  5.56%

🚧 UPSCAYL_PROGRESS:  5.56%

🐞 BACKEND REPORTED:  image upscayl:  8.33%

🚧 UPSCAYL_PROGRESS:  8.33%

🐞 BACKEND REPORTED:  image upscayl:  11.11%

🚧 UPSCAYL_PROGRESS:  11.11%

🐞 BACKEND REPORTED:  image upscayl:  13.89%

🚧 UPSCAYL_PROGRESS:  13.89%

🐞 BACKEND REPORTED:  image upscayl:  16.67%

🚧 UPSCAYL_PROGRESS:  16.67%

🐞 BACKEND REPORTED:  image upscayl:  19.44%

🚧 UPSCAYL_PROGRESS:  19.44%

🐞 BACKEND REPORTED:  image upscayl:  22.22%

🚧 UPSCAYL_PROGRESS:  22.22%

🐞 BACKEND REPORTED:  image upscayl:  25.00%

🚧 UPSCAYL_PROGRESS:  25.00%

🐞 BACKEND REPORTED:  image upscayl:  27.78%

🚧 UPSCAYL_PROGRESS:  27.78%

🐞 BACKEND REPORTED:  image upscayl:  30.56%

🚧 UPSCAYL_PROGRESS:  30.56%

🐞 BACKEND REPORTED:  image upscayl:  33.33%

🚧 UPSCAYL_PROGRESS:  33.33%

🐞 BACKEND REPORTED:  image upscayl:  36.11%

🚧 UPSCAYL_PROGRESS:  36.11%

🐞 BACKEND REPORTED:  image upscayl:  38.89%

🚧 UPSCAYL_PROGRESS:  38.89%

🐞 BACKEND REPORTED:  image upscayl:  41.67%

🚧 UPSCAYL_PROGRESS:  41.67%

🐞 BACKEND REPORTED:  image upscayl:  44.44%

🚧 UPSCAYL_PROGRESS:  44.44%

🐞 BACKEND REPORTED:  image upscayl:  47.22%

🚧 UPSCAYL_PROGRESS:  47.22%

🐞 BACKEND REPORTED:  image upscayl:  50.00%

🚧 UPSCAYL_PROGRESS:  50.00%

🐞 BACKEND REPORTED:  image upscayl:  52.78%

🚧 UPSCAYL_PROGRESS:  52.78%

🐞 BACKEND REPORTED:  image upscayl:  55.56%

🚧 UPSCAYL_PROGRESS:  55.56%

🐞 BACKEND REPORTED:  image upscayl:  58.33%

🚧 UPSCAYL_PROGRESS:  58.33%

🐞 BACKEND REPORTED:  image upscayl:  61.11%

🚧 UPSCAYL_PROGRESS:  61.11%

🐞 BACKEND REPORTED:  image upscayl:  63.89%

🚧 UPSCAYL_PROGRESS:  63.89%

🐞 BACKEND REPORTED:  image upscayl:  66.67%

🚧 UPSCAYL_PROGRESS:  66.67%

🐞 BACKEND REPORTED:  image upscayl:  69.44%

🚧 UPSCAYL_PROGRESS:  69.44%

🐞 BACKEND REPORTED:  image upscayl:  72.22%

🚧 UPSCAYL_PROGRESS:  72.22%

🐞 BACKEND REPORTED:  image upscayl:  75.00%

🚧 UPSCAYL_PROGRESS:  75.00%

🐞 BACKEND REPORTED:  image upscayl:  77.78%

🚧 UPSCAYL_PROGRESS:  77.78%

🐞 BACKEND REPORTED:  image upscayl:  80.56%

🚧 UPSCAYL_PROGRESS:  80.56%

🐞 BACKEND REPORTED:  image upscayl:  83.33%

🚧 UPSCAYL_PROGRESS:  83.33%

🐞 BACKEND REPORTED:  image upscayl:  86.11%

🚧 UPSCAYL_PROGRESS:  86.11%

🐞 BACKEND REPORTED:  image upscayl:  88.89%

🚧 UPSCAYL_PROGRESS:  88.89%

🐞 BACKEND REPORTED:  image upscayl:  91.67%

🚧 UPSCAYL_PROGRESS:  91.67%

🐞 BACKEND REPORTED:  image upscayl:  94.44%

🚧 UPSCAYL_PROGRESS:  94.44%

🐞 BACKEND REPORTED:  image upscayl:  97.22%

🚧 UPSCAYL_PROGRESS:  97.22%

🐞 BACKEND REPORTED:  image upscayl:  Upscayl Successful

🚧 UPSCAYL_PROGRESS:  Upscayl Successful

🐞 BACKEND REPORTED:  💯 Done upscaling
🐞 BACKEND REPORTED:  ♻ Scaling and converting now...
🐞 BACKEND REPORTED:  🚫 Skipping scaling and converting
upscaledImagePath:  /home/graphix/Downloads/3e7sbl2v59ea1_upscayl_4x_SwatKats.png
💯 UPSCAYL_DONE:  /home/graphix/Downloads/3e7sbl2v59ea1_upscayl_4x_SwatKats.png
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"SwatKats","value":"SwatKats"}
⚙️ Getting gpuId from localStorage:  1
⚙️ Getting rememberOutputFolder from localStorage:  false
```

**Comment by NayamAmarshe** [comment]
The model name must contain the scale parameter too! x2

**Comment by aaronliu0130** [comment]
Or, you could set your scale option in settings to 2 instead of 4.

**Comment by NayamAmarshe** [comment]
> Or, you could set your scale option in settings to 2 instead of 4.

No that might not work.

**Comment by reginaldbuxley** [comment]
I started from scratch and converted my the example model in chaiNNer from scratch and gave it the same name as the model's file name (2x_SwatKats_154000_G). This resulted in the same type of issue:

![image](https://github.com/upscayl/upscayl/assets/24417317/fe92f817-1a8a-42ee-9680-e3f684fd6201)

So then based on what @NayamAmarshe posted regarding includes("x2") I performed another fresh convert, this time naming the model x2_SwatKats_154000_G:

![image](https://github.com/upscayl/upscayl/assets/24417317/e5a3a4c1-0624-4c47-9462-9593a27d8d60)

At this point, it does seem to work. So I tried it with another model I was having an issue with:

![image](https://github.com/upscayl/upscayl/assets/24417317/fd1a78da-0309-4ea1-b3c5-dcf8fb06abe1)

Again - this seems to have fixed the issue. So I will close this issue at this time. **However** - I must imagine that setting the initial scale by looking at the name of the model, can not be the best way to handle this decision. Is there not a way to detect the scale value of a model once it has been converted to NCNN as there is when it is in PTH  format?

![image](https://github.com/upscayl/upscayl/assets/24417317/051bdee7-f1a9-4325-b6a7-e66354f1eba9)
